util/feedback.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+from starlette.exceptions import HTTPException
+from config.db import get_database
+import logging
+DB_NAME = "underline"
+
+# Given an event id and feedback id, attempt to delete the feedback from the event's comment_ids array
+async def delete_feedback(event_id, feedback_id):
+    db = get_database()
+    
+    column = db[DB_NAME]["events"]
+
+    found_event = column.find_one({"_id": event_id})
+    
+    if not found_event:
+        raise HTTPException(status_code=400,
+                            detail="Event ID is invalid")
+
+
+    if feedback_id in found_event["comment_ids"]:
+        found_event["comment_ids"].remove(feedback_id)
```

Comment:
the name of the field was changes recently but its fine to leave as is in the PR, once pulled i'll patch it over